ratis-shell/src/main/libexec/ratis-shell-config.sh

```diff
@@ -58,6 +59,16 @@ if [[ ${JAVA_MAJORMINOR} != 001008 && ${JAVA_MAJOR} != 011 ]]; then
   exit 1
 fi
 
+local RATIS_SHELL_CLASSPATH
+
+while read -d '' -r jarfile ; do
+    if [[ "$RATIS_SHELL_CLASSPATH" == "" ]]; then
+        RATIS_SHELL_CLASSPATH="$jarfile";
+    else
+        RATIS_SHELL_CLASSPATH="$RATIS_SHELL_CLASSPATH":"$jarfile"
+    fi
+done < <(find "$RATIS_SHELL_LIB_DIR" ! -type d -name '*.jar' -print0 | sort -z)
```

Comment:
There is a syntax error:
```shell
$sh ./bin/ratis -e
/Users/szetszwo/Downloads/2.3.0/apache-ratis-3.0.0-SNAPSHOT-bin/bin/../libexec/ratis-shell-config.sh: line 70: syntax error near unexpected token `<'
/Users/szetszwo/Downloads/2.3.0/apache-ratis-3.0.0-SNAPSHOT-bin/bin/../libexec/ratis-shell-config.sh: line 70: `done < <(find "$RATIS_SHELL_LIB_DIR" ! -type d -name '*.jar' -print0 | sort -z)'
Unsupported command -e
Usage: ratis COMMAND [GENERIC_COMMAND_OPTIONS] [COMMAND_ARGS]

COMMAND is one of:
-e   sh    	 Command line tool for ratis

GENERIC_COMMAND_OPTIONS supports:
-e   -D<property=value>	 Use a value for a given ratis-shell property

Commands print help when invoked without parameters.
```

Reply:
hi @szetszwo, I test several commands, `bash` is different from `sh`.
|  command   | result  |
|  ----  | ----  |
| /bin/bash ./bin/ratis  | ok |
| /bin/bash ./bin/ratis -e | ok |
| /bin/bash ./bin/ratis sh | ok |
| /bin/sh ./bin/ratis sh | error |
| /bin/sh ./bin/ratis -e | error |

We should use `bash` as the same as the head in script files.

![image](https://user-images.githubusercontent.com/95013770/184310774-67b00f61-0528-4d88-bc9b-4fb24ae4d4db.png)

Reply:
@leo65535 , bash worked file.  Thanks!

It seems that tar somehow have not preserved the permissions
```shell
$tar tvzf ratis-assembly/target/apache-ratis-3.0.0-SNAPSHOT-bin.tar.gz
...
-rw-r--r--  0 root   root     2152 Jan 22  2020 apache-ratis-3.0.0-SNAPSHOT-bin/bin/ratis
-rw-r--r--  0 root   root     1580 Jan 22  2020 apache-ratis-3.0.0-SNAPSHOT-bin/conf/log4j.properties
-rw-r--r--  0 root   root     7118 Jan 22  2020 apache-ratis-3.0.0-SNAPSHOT-bin/examples/README.md
-rw-r--r--  0 root   root     1138 Jan 22  2020 apache-ratis-3.0.0-SNAPSHOT-bin/examples/bin/client.sh
-rw-r--r--  0 root   root     1859 Jan 22  2020 apache-ratis-3.0.0-SNAPSHOT-bin/examples/bin/common.sh
-rw-r--r--  0 root   root      921 Jan 22  2020 apache-ratis-3.0.0-SNAPSHOT-bin/examples/bin/server.sh
-rw-r--r--  0 root   root     1598 Jan 22  2020 apache-ratis-3.0.0-SNAPSHOT-bin/examples/bin/start-all.sh
-rw-r--r--  0 root   root      922 Jan 22  2020 apache-ratis-3.0.0-SNAPSHOT-bin/examples/bin/stop-all.sh
...
```
The permissions are correct in ratis-assembly/target dir
```shell
$ls -l ratis-assembly/target/apache-ratis-3.0.0-SNAPSHOT-bin/apache-ratis-3.0.0-SNAPSHOT-bin/bin/
total 8
-rwxr-xr-x  1 szetszwo  staff  2152 Jan 22  2020 ratis
$ls -l ratis-assembly/target/apache-ratis-3.0.0-SNAPSHOT-bin/apache-ratis-3.0.0-SNAPSHOT-bin/examples/bin/
total 40
-rwxr-xr-x  1 szetszwo  staff  1138 Jan 22  2020 client.sh
-rwxr-xr-x  1 szetszwo  staff  1859 Jan 22  2020 common.sh
-rwxr-xr-x  1 szetszwo  staff   921 Jan 22  2020 server.sh
-rwxr-xr-x  1 szetszwo  staff  1598 Jan 22  2020 start-all.sh
-rwxr-xr-x  1 szetszwo  staff   922 Jan 22  2020 stop-all.sh
```